src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig

```diff
@@ -1342,3 +1352,36 @@
     {{- block('submittable_input_button_widget') -}}
   </div>
 {% endblock %}
+
+{% block accordion_widget %}
+  <div class="accordion accordion-form" id="{{ form.vars.id }}">
+    {% set firstChild = true %}
+    {% for child in form %}
+      {% if child.vars.compound %}
+      {% set isExpanded = firstChild and expand_first %}
+      <div class="card">
+        <div class="card-header" id="{{ child.vars.id }}_accordion_header">
+          <h2 class="mb-0">
+            <button class="accordion-form-button btn btn-block text-left{% if not isExpanded %} collapsed{% endif %}" type="button" data-toggle="collapse" data-target="#{{ child.vars.id }}_accordion" aria-expanded="{% if isExpanded %}true{% else %}false{% endif %}" aria-controls="{{ child.vars.id }}_accordion">
```

Comment:
CI accepted this line 😮

Reply:
I don't think we have max columns conditions on twig But I can improve the readability of this one a bit I guess